specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
Ah yes, and in terms of whether it is contained or associated, I don't have an opinion offhand.